examples/flat-serialisation/anonymous-owner.json

```diff
@@ -0,0 +1,50 @@
+{
+  "version": "0.1",
+  "statementGroups": [
+    {
+      "statementID": "32e09b90-203d-42d1-bd77-a6465bca2552",
+      "beneficialOwnershipStatements": [
+        {
+          "statementID": "0e5c86db-f2b0-4e11-bff0-db9eee3db221",
+          "statementDate": "2017-07-18",
+          "subject": {
```

Comment:
Do we need the complex structure here? Why subject rather than entity?

Reply:
This will have the same structure as interestedParty, with more than just entities as possible targets of possible beneficial ownership statements. We will need 'arrangement', at least, to describe trusts and joint ventures.